py/dml/structure.py

```diff
@@ -1457,7 +1479,7 @@ def process_method_implementations(obj, name, implementations,
 # Second stage of object creation, protected by error context
 # TODO: we should probably split up mkobj into several methods with
 # clearer responsibilities
-def mkobj2(obj, obj_specs, params, each_stmts):
+def mkobj2(obj, obj_specs, params, each_stmts, used_templates):
```

Comment:
Oh, I don't use `used_templates` anymore as part of `mkobj2`, only `mkobj`. The parameter for it can be tossed away.